app/src/ui/app-error.tsx

```diff
@@ -122,7 +125,7 @@ export class AppError extends React.Component<IAppErrorProps, IAppErrorState> {
 
     const className = monospace ? 'monospace' : undefined
 
-    return <p className={className}>{e.message}</p>
+    return <p className={className}>{this.formatGitErrorMessage(e.message)}</p>
```

Comment:
question: do we want to call the format function for any kind of error or only for error messages that come from the output of `git` (since we only use monospaced fonts in that scenario)?

Reply:
Ah, yeah good call. It shouldn't matter since the only added cost would be one `.indexOf` call but I agree with you that we should keep these two concepts aligned.

Reply:
Fixed in https://github.com/desktop/desktop/pull/9945/commits/ce210cacf8d86e478fff32f1e2ef46236580ed3a